src/OmniSharp.Roslyn.CSharp/Helpers/QuickFixExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using OmniSharp.Models;
+
+namespace OmniSharp.Helpers
+{
+    internal static class QuickFixExtensions
+    {
+        internal static async Task AddQuickFix(this ICollection<QuickFix> quickFixes, OmniSharpWorkspace workspace, Location location)
+        {
+            if (location.IsInSource)
+            {
+                var quickFix = await QuickFixHelper.GetQuickFix(workspace, location);
+                quickFixes.Add(quickFix);
+            }
+        }
+
+        internal static async Task AddQuickFixes(this ICollection<QuickFix> quickFixes, OmniSharpWorkspace workspace, IEnumerable<ISymbol> symbols)
```

Comment:
If you make the rename above, this could be `AddRange`. Also, does it seem better to swap the last two parameters?